pipeline/07-export.R

```diff
@@ -105,22 +105,23 @@ assessment_pin_prepped <- assessment_pin %>%
   ) %>%
   select(
     township_code, meta_pin, meta_class, meta_nbhd_code,
-    property_full_address, loc_cook_municipality_name, meta_pin10,
+    property_full_address, loc_tax_municipality_name, meta_pin10,
     meta_tieback_key_pin, meta_tieback_proration_rate,
     prior_near_land, prior_near_bldg, prior_near_tot,
     prior_near_land_rate, prior_near_bldg_rate, prior_near_land_pct_total,
     pred_pin_final_fmv, pred_pin_final_fmv_land, pred_pin_final_fmv_bldg,
     pred_pin_final_fmv_round, land_rate_per_sqft, pred_pin_land_rate_effective,
     pred_pin_bldg_rate_effective, pred_pin_land_pct_total,
     prior_near_yoy_change_nom, prior_near_yoy_change_pct,
-    sale_recent_1_date, sale_recent_1_price, sale_recent_1_document_num,
-    sale_recent_2_date, sale_recent_2_price, sale_recent_2_document_num,
+    sale_recent_1_date, sale_recent_1_price,
+    sale_recent_1_outlier_type, sale_recent_1_document_num,
+    sale_recent_2_date, sale_recent_2_price,
+    sale_recent_2_outlier_type, sale_recent_2_document_num,
     char_yrblt, char_total_bldg_sf, char_type_resd, char_land_sf,
     char_unit_sf, flag_nonlivable_space, flag_pin10_5yr_num_sale,
     flag_common_area, flag_proration_sum_not_1, flag_pin_is_multiland,
-    flag_land_gte_95_percentile, flag_bldg_gte_95_percentile,
+    flag_land_gte_95_percentile,
     flag_land_value_capped, flag_prior_near_to_pred_unchanged,
-    flag_pred_initial_to_final_changed,
```

Comment:
`flag_bldg_gte_95_percentile` and `flag_pred_initial_to_final_changed` appear not to be present anymore in `assessment_pin`, so I removed them from the workbook flags. They also didn't seem to be present in the workbook template anyway.